app/assets/stylesheets/refinery/home.scss

```diff
@@ -33,10 +37,23 @@
 
       cursor: default;
       background-image: image-url('bike-lane.jpg');
-      transition: .5s ease-in-out;
+      transition: 2s ease-in-out;
 
       h1 { margin: 0 0 .5rem 0; }
 
+      &__background-image {
+        position: absolute;
+        top: 0;
+        left: 0;
+
+         height: 100%;
+        width: 100%;
```

Comment:
I think that lines 45-50 can be removed since they were implemented in the `.styled-box` Sass file, lines 36-42.